lib/init.coffee

```diff
@@ -252,14 +252,17 @@ module.exports =
           messages = @parseSchemaMessages(textEditor, output)
         if messages.length
           for message in messages
-            message.type = 'Error'
-            message.text = message.text + ' (' + schemaUrl + ')'
-            message.filePath = textEditor.getPath()
+            message.severity = 'error'
+            message.excerpt = message.excerpt + ' (' + schemaUrl + ')'
```

Comment:
~~Here we go: 22ba410e17589004189cc4af4a8567ca34dc72a7~~

And hopefully making the linter happy: 0951606aae680872e27d1279e8b35c6d88d482c6